src/main/java/io/jaegertracing/tests/ISpanCounter.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2018 The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.tests;
+
+import java.io.Closeable;
+
+/**
+ * @author Pavol Loffay
+ */
+public interface ISpanCounter extends Closeable {
```

Comment:
Adding I to interfaces is common in C#, not sure if in java.

Reply:
I'm also not sure about the standard way :)